Jenkinsfile

```
@@ -278,7 +299,11 @@ pipeline {
 											done
 										fi
 									fi
-									cp *.$binariesExtension "${WORKSPACE}/eclipse.platform.swt/binaries/org.eclipse.swt.${PLATFORM}/"
+									if [[ ${PLATFORM} == gtk4.linux.x86_64 ]]; then
+										cp libswt-pi4-gtk*.so "${WORKSPACE}/eclipse.platform.swt/binaries/org.eclipse.swt.${DEST_PLATFORM}/"
+									else
+										cp *.$binariesExtension "${WORKSPACE}/eclipse.platform.swt/binaries/org.eclipse.swt.${DEST_PLATFORM}/"
+									fi
```

Couldn't we implement this without the extra `DEST_PLATFORM` variable and therefore slightly simpler?
```suggestion
									if [[ ${PLATFORM} == gtk4.linux.x86_64 ]]; then
										cp libswt-pi4-gtk*.so "${WORKSPACE}/eclipse.platform.swt/binaries/org.eclipse.swt.gtk.linux.x86_64/"
									else
										cp *.$binariesExtension "${WORKSPACE}/eclipse.platform.swt/binaries/org.eclipse.swt.${PLATFORM}/"
									fi
```